pkg/controller/user/create.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package user
+
+import (
+	"net/http"
+
+	"github.com/google/exposure-notifications-verification-server/pkg/controller"
+	"github.com/google/exposure-notifications-verification-server/pkg/controller/flash"
+)
+
+func (c *Controller) HandleCreate() http.Handler {
+	type FormData struct {
+		Email    string `form:"email,required"`
+		Name     string `form:"name,required"`
+		Admin    bool   `form:"admin"`
+		Disabled bool   `form:"disabled"`
```

Comment:
I dropped this from the page as it didn't make sense to create a user in disabled state.